_includes/activity/github-desktop/02-clone-locally.md

```diff
@@ -4,5 +4,5 @@ After you've created a repository on the remote, the next step is to clone it to
 1. On GitHub.com, navigate to the Code tab of the repository.
 1. On the right side of the screen, click `Clone or download`.
```

Comment:
@crichid I changed this line previously because the screenshot does not show this procedure, instead it copies the clone URL.